src/plugin.ts

```diff
@@ -20,8 +20,6 @@ export async function plugin(inputs: PluginInputs, env: Env) {
     logger: new Logs("debug"),
   };
 
-  context.logger.info(
-    `Will check the following repos / orgs: [${context.config.repos.monitor.join(", ")}] and ignore [${context.config.repos.ignore.join(", ")}]`
-  );
+  context.logger.info(`Will check the following repos: [${context.config.repos.monitor.join(", ")}] and ignore [${context.config.repos.ignore.join(", ")}]`);
```

Comment:
Is it maybe cleaner to log something like `("Will check the following repos:", {repos: config.repos})`? I mean cleaner/easier to read in the log itself

Reply:
https://github.com/Meniole/automated-merging/actions/runs/10501574194/job/29091781547#step:6:34